packages/cerebral-provider-forms/src/form.js

```diff
@@ -61,6 +61,9 @@ export default function computedForm (formValueTag) {
   return compute(
     formValueTag,
     (formValue) => {
+      if (typeof formValue !== 'object') {
```

Comment:
Hm, yeah, good point... if it is not an object, just return the value directly. That way it is easy to debug as well.. cause problems will occur in the component, where you try to access something that is not there :)